source/includes/gcp/_load_balancer.md

```diff
@@ -1,3 +1,59 @@
 #### Load balancer
 
 HTTP(S) load balancing can balance HTTP and HTTPS traffic across multiple backend instances, across multiple regions. Your entire app is available via a single global IP address, resulting in a simplified DNS setup. HTTP(S) load balancing is scalable, fault-tolerant, requires no pre-warming, and enables content-based load balancing. For HTTPS traffic, it provides SSL termination and load balancing.
+
+<!-------------------- LIST LOAD BALANCERS -------------------->
+
+#### List instance groups
+
+```shell
+curl -X GET \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/gcp/test-area/loadbalancers"
+
+# The above command returns JSON structured like this:
+```
+
+```json
+{
+    "data": [
+        {
+            "id": "5538770461112637304",
+            "forwardingRule": "https://www.googleapis.com/compute/v1/projects/cmc-gcp-htu/global/forwardingRules/forward123",
+            "targetProxy": "https://www.googleapis.com/compute/v1/projects/cmc-gcp-htu/global/targetHttpsProxies/hellowrold-target-proxy",
```

Comment:
maybe fix `hellowrold` -> `helloworld` or use a better name for it `my-target-proxy` for example